### PR TITLE
release: 0.12.0

### PR DESCRIPTION
**Pull request by mistydemeo**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,48 @@
...
+
+### PowerShell installer tests
+
+We now run PowerShell installers end to end in cargo-dist's own tests.
+
+* impl @Gankra [chore: add ruin_me powershell tests](https://github.com/axodotdev/cargo-dist/pull/862)
+
+### Invoke `rustup target add` for additional forms of cross-compiling
+
+We now run `rustup target add` unconditionally when cross-compiling; before, this was limited to macOS or for musl Linux.
```
Comment (ashleygwilliams) [inline CHANGELOG.md:32]: it might be nice to explain why this is cool

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,48 @@
...
+
+* impl @AustinWise [feat: always rustup target add in cross](https://github.com/axodotdev/cargo-dist/pull/846)
+
+### Allow overriding install path in shell and PowerShell installers
+
+Although some installer configuration allows setting the installation path, there was previously no way to force the installer to install to a location of the user's choosing. We've added a feature to do so via the new `CARGO_DIST_FORCE_INSTALL_DIR` environment variable.
```
Comment (ashleygwilliams) [inline CHANGELOG.md:38]: maybe we can elaborate the use case here? (unless it is marginal...)

**Review by ashleygwilliams — APPROVED** [review]
approved, some small comments on places we could elaborate but mostly looks good!!

File: CHANGELOG.md (hunk 2 above)
Comment (mistydemeo) [reply CHANGELOG.md:38]: It's currently intended for our internal use, so I'll say that directly.

**Pull request merged**